lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -335,17 +334,22 @@ def create_tar(cv, cvv, repositories, json)
 
             `tar cvfh #{repo_tar} #{repo_dirs.join(" ")}`
           end
-        end
 
-        Dir.chdir("#{export_dir}/#{export_prefix}") do
-          File.open(export_file, 'w') do |file|
-            file.write(JSON.pretty_generate(json))
+          Dir.chdir("#{export_dir}/#{export_prefix}") do
+            File.open(export_file, 'w') do |file|
+              file.write(JSON.pretty_generate(json))
+            end
           end
-        end
 
-        Dir.chdir(export_dir) do
-          `tar cf #{export_tar} #{export_prefix}`
-          FileUtils.rm_rf(export_prefix)
+          Dir.chdir(export_dir) do
+            `tar cf #{export_tar} #{export_prefix}`
+            FileUtils.rm_rf(export_prefix)
+          end
+
+        elsif cv['composite']
+          puts "Cannot export composite content view."
```

Comment:
Definitely not true - composite CVs are exported without repositories, just the json.